timemachine/fe/single_topology.py

```diff
@@ -325,6 +295,13 @@ def setup_dummy_interactions(
     return bonded_idxs, bonded_params
 
 
+def canonicalize_bonds(bonds: NDArray[np.int32]) -> NDArray[np.int32]:
+    assert bonds.ndim == 2
```

Comment:
nice - good idea to batch this! (would be useful in other parts of the code later on for angles, torsions etc) too. (fine as-is for this PR).